orderable/managers.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class OrderableManager(models.Manager):
+
```

Comment:
(A docstring would be nice here actually, what with this being a library and all)